data/teamData.js

```diff
@@ -116,19 +105,19 @@ export default {
       linkedin: "https://www.linkedin.com/in/anisharao14/"
     },
     {
-      name: "Aryn Harmon",
+      name: "Neeraj Aggarwal",
       role: "Software Developer",
```

Comment:
I think this should be changed to reflect his most recent position

Reply:
I think this is correct? I updated everything for the fall